.github/workflows/molecule.yml

```diff
@@ -8,34 +8,33 @@ on:
 jobs:
 
   list-scenarios:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     outputs:
       matrix: ${{ steps.listscenarios.outputs.scenarios }}
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - id: listscenarios
         uses: ome/action-ansible-molecule-list-scenarios@main
 
   test:
     name: Test
     needs:
       - list-scenarios
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     strategy:
       # Keep running so we can see if other tests pass
       fail-fast: false
       matrix:
         scenario: ${{fromJson(needs.list-scenarios.outputs.matrix)}}
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - uses: actions/setup-python@v4
```

Comment:
``actions/setup-python`` bump to v5

Reply:
fixed in https://github.com/ome/prod-playbooks/pull/379/commits/fee1fdea5cbc1687f0ad82ef8205ba96d1d223ad